app/views/users/phone_setup/index.html.erb

```diff
@@ -49,3 +49,11 @@
 <% end %>
 
 <%= render 'shared/cancel_or_back_to_options' %>
+<% if FeatureManagement.phone_recaptcha_enabled? %>
```

Comment:
Something I wanted to note, all users currently would see the disclaimer unless recaptcha is disabled, regardless of international or not. Wondering if its worth to dynamically load this disclaimer based on what the user selects or the type of phone number entered in. would that be overly complicated?

Reply:
> Wondering if its worth to dynamically load this disclaimer based on what the user selects or the type of phone number entered in. would that be overly complicated?

I wondered about this too. I had a similar Slack conversation with @2hsieh about this a week or so back, where he made a good point that we should only tell the user that they are subject to terms when those terms would actually apply. So I think it'd be good to try to make it so the disclaimer is only visible when an international number is selected.

Reply:
Although, given the technical complexity of that and the potential risks associated with dynamically adding and removing the disclaimer, I could personally be convinced to go with the simpler option here.

Reply:
I think for now, after disccusing with Andrew D and Andrew H we will keep it appearing for all users when recaptcha is enabled.